pkg/fftm/manager.go

```diff
@@ -105,10 +105,6 @@ type manager struct {
 func InitConfig() {
 	tmconfig.Reset()
 	events.InitDefaults()
-
-	if config.GetBool(tmconfig.MetricsEnabled) {
-		metrics.Registry()
```

Comment:
Won't removing this disable metrics?

Reply:
I re-tested before and after and think I came to the conclusion this was extraneous. `tmconfig.Reset()` sets `MetricsEnabled` to `false` (see https://github.com/hyperledger/firefly-transaction-manager/blob/574b5c73105016c8b6bcd24ff5fefb5858bea7b3/pkg/fftm/manager.go#L106) but now I'm questioning it so let me check.

Reply:
So looking at the code and re-re-testing with this commit in place, I think this _was_ extraneous code. I think

https://github.com/hyperledger/firefly-transaction-manager/blob/574b5c73105016c8b6bcd24ff5fefb5858bea7b3/pkg/fftm/manager.go#L173

performs the same registry initialisation that was being performed here.

Reply:
Super, thanks for checking that 👍